packages/iframe-stamper/src/index.ts

```diff
@@ -13,16 +13,22 @@ export enum IframeEventType {
   InjectCredentialBundle = "INJECT_CREDENTIAL_BUNDLE",
   // Event sent by the parent to inject a private key export bundle into the iframe.
   // Value: the bundle to inject
+  // Key Format (optional): the key format to encode the private key in after it's exported and decrypted: HEXADECIMAL or SOLANA. Defaults to HEXADECIMAL.
+  // Public Key (optional): the public key of the exported private key. Required when the key format is SOLANA.
   InjectKeyExportBundle = "INJECT_KEY_EXPORT_BUNDLE",
   // Event sent by the parent to inject a wallet export bundle into the iframe.
   // Value: the bundle to inject
   InjectWalletExportBundle = "INJECT_WALLET_EXPORT_BUNDLE",
   // Event sent by the parent to inject an import bundle into the iframe.
   // Value: the bundle to inject
   InjectImportBundle = "INJECT_IMPORT_BUNDLE",
-  // Event sent by the parent to extract an encrypted bundle from the iframe.
-  // Value: the bundle to inject
+  // Event sent by the parent to extract an encrypted wallet bundle from the iframe.
+  // Value: none
   ExtractWalletEncryptedBundle = "EXTRACT_WALLET_ENCRYPTED_BUNDLE",
+  // Event sent by the parent to extract an encrypted private key bundle from the iframe.
+  // Value: none
+  // Key Format (optional): the key format to decode the private key in before it's encrypted for import: HEXADECIMAL or SOLANA. Defaults to HEXADECIMAL.
```

Comment:
what is the "key format" here? Is this the return value? Or an input?

Reply:
an input